cmd/network/evpn-utils.go

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022-2023 Intel Corporation, or its subsidiaries.
+// Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
+// Copyright (c) 2024 Ericsson AB.
+
+// Package network implements the network related CLI commands
+package network
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/PraserX/ipconv"
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+	pc "github.com/opiproject/opi-api/network/opinetcommon/v1alpha1/gen/go"
+)
+
+// PrintComponents prints the components with their details
+func PrintComponents(comp []*pb.Component) string {
```

Comment:
This function does not print anything. What it does is to get together information about Components. Maybe rename to `ComposeComponentsInfo` or print directly in the function?